torch/utils/_config_module.py

```diff
@@ -255,7 +274,13 @@ def load_config(self, maybe_pickled_config: Union[bytes, Dict[str, Any]]) -> Non
             config = pickle.loads(maybe_pickled_config)
         else:
             config = maybe_pickled_config
-        self._config.update(config)
+        for k, v in config.items():
+            if k in self._config:
+                self.__setattr__(k, v)
```

Comment:
the nit for getattr applies for setattr too! (use setattr(self, k, v) instead of calling magic method)

Reply:
Done.